connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -281,45 +274,38 @@ public void testDestroyConnector() throws Exception {
         } catch (ExecutionException e) {
             assertTrue(e.getCause() instanceof NotFoundException);
         }
```

Comment:
Resolved and pushed.

Reply:
The second line from my suggestion is still missing (which is important since the cause of the `ExecutionException` could be anything right now, and we need to assert that it's a `NotFoundException` instead).

Reply:
Thanks for the pickup, I have rebased the PR with the fix (see https://github.com/apache/kafka/pull/12728/commits/784063310fe4676a8c57b73a566985ece25d0bc7). Resolve the conversation if its all good.

Reply:
Resolving this as fixed, feel free to unresolve if there are still problems.